pg-index-health/src/main/java/io/github/mfvanek/pg/checks/host/NotValidConstraintsOnHost.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019-2024. Ivan Vakhrushev and others.
+ * https://github.com/mfvanek/pg-index-health
+ *
+ * This file is a part of "pg-index-health" - a Java library for
+ * analyzing and maintaining indexes health in PostgreSQL databases.
+ *
+ * Licensed under the Apache License 2.0
+ */
+
+package io.github.mfvanek.pg.checks.host;
+
+import io.github.mfvanek.pg.common.maintenance.Diagnostic;
+import io.github.mfvanek.pg.connection.PgConnection;
+import io.github.mfvanek.pg.model.PgContext;
+import io.github.mfvanek.pg.model.constraint.Constraint;
+import io.github.mfvanek.pg.model.constraint.ConstraintType;
+
+import java.util.List;
+import javax.annotation.Nonnull;
+
+public class NotValidConstraintsOnHost extends AbstractCheckOnHost<Constraint> {
```

Comment:
Please rename to `NotValidConstraintsCheckOnHost`.  
See https://github.com/mfvanek/pg-index-health/pull/374#discussion_r1581845619